Sources/WebDriver/Session.swift

```diff
@@ -321,6 +328,11 @@ public class Session {
         try webDriver.send(Requests.SessionWindowSize.Post(session: id, windowHandle: handle, width: width, height: height))
     }
 
+    /// Set the current geolocation 
+    public func setLocation(latitude: Double, longitude: Double, altitude: Double, location: Location) throws {
```

Comment:
The first three params are redundant with the last. This should be

```swift
public func setLocation(_ location: Location) throws {
```

and if you'd like, also an overload:

```swift
public func setLocation(latitude: Double, longitude: Double, altitude: Double) throws {
    try setLocation(Location(latitude: latitude, longitude: longitude, altitude: altitude))
}
```